### ISSUE TITLE
any idea on multi batch training

### ISSUE THREAD

**Issue by qinglintian** [smallcorgi/Faster-RCNN_TF#54]
same as faster rcnn, the training only allow single image per batch in training. In caffe, the parameter of "iter_size" can be adjusted to do multi batch training as weights are updated after "iter_size" iterations, i.e., images. Can this be done in TF?

Thank you.

**Comment by qinglintian** [comment]
following http://stackoverflow.com/questions/37710974/how-to-accumulate-and-appy-gradients-for-async-n-step-dqnetwork-update-in-tensor, I modified train.py in most recent codes of this project, using TensorFlow 1.0 on customized dataset. The basic idea is create a placeholder as the switch to apply the accumulated gradients over several iteration.
But after several iterations, a warning appears indicating overflow in exp function in lib/fast_rcnn/bbox_transform.py, line 49.
https://github.com/rbgirshick/py-faster-rcnn/issues/65 this thread also reports overflow in exp in bbox_transform. I followed the solutions there by trying to adjust learning_rate (as low as 0.000001), RNG_SEED with no success. The annotation of the bounding-box is also 1-based.

@smallcorgi do you have any idea in implementing multi batch training? please help, thank you.
